grommet/grommet
# fix(Select): clean search field when close dropdown

#### What does this PR do?
It cleans search field in select component when closes without selection any option

#### Where should the reviewer start?
Start on Select component 

#### How should this be manually tested?
1. Start storybook
2. Open Input/Select/Search
3. Open the first dropdown and search by option 3
4. Click out of dropdown, it should close
5. Open again the dropdown and the search field must be empty

#### What are the relevant issues?
https://github.com/grommet/grommet/issues/5865

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/7703431/147111988-56c99398-312e-46f8-ab13-2804292ccc25.png)
![image](https://user-images.githubusercontent.com/7703431/147112036-b7578bec-6c5a-4a02-9d26-dcf978d002f2.png)


#### Is this change backwards compatible or is it a breaking change?
Yes, it is.
